packages/taquito-michel-codec/src/michelson-types.ts

```diff
@@ -413,7 +414,8 @@ const protoLevel: Record<ProtocolID, number> = {
   Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A: 12,
   PtJakartaiDz69SfDDLXJSiuZqTSeSKRDbKVZC8MNzJnvRjvnGw: 13,
   PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY: 13,
-  ProtoALphaALphaALphaALphaALphaALphaALphaALphaDdp3zK: 14,
+  PtKathmanYPEsBau83zRwXK19HAQRzZSj6pCXnvtyLZFjRxdQxt: 14,
```

Comment:
It should be PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg.

Reply:
fixed